js/views/userPage/TypeFilter.js

```diff
@@ -0,0 +1,56 @@
+/* categoryFilter.js and typeFilter.js share quite a bit. Ensure that when one is updated,
+the other is also maintained.
+*/
+
+import $ from 'jquery';
+import loadTemplate from '../../utils/loadTemplate';
+import BaseVw from '../baseVw';
+
+export default class extends BaseVw {
+  constructor(options = {}) {
+    const opts = {
+      ...options,
+    };
+
+    opts.initialState = {
```

Comment:
Just as an FYI, you could have included the `initialState` key inside the `opts` declaration above (after the `...options`).

Reply:
That makes sense. Do you think it might be less clean that way though? 🤔

Reply:
I think it's cleaner, but i imagine it's a matter of taste... so totally up to you.